docs/content/docs/dev/table/sqlClient.md

```diff
@@ -54,8 +57,18 @@ or explicitly use `embedded` mode:
 
 ```bash
 ./bin/sql-client.sh embedded
+``` 
+
+For gateway mode, you can start the CLI by calling:
+
+```bash
+./bin/sql-client.sh gateway --endpoint <gateway address>
 ```
 
+In the gateway mode, the CLI submits the SQL to the specified remote gateway to execute statements.
+
+<span class="label label-danger">Note</span> SQL Client only supports to connect to the [REST Endpoint]({{< ref "docs/dev/table/sql-gateway/rest" >}}#rest-api) whose version is larger than v1.
```

Comment:
```suggestion
<span class="label label-danger">Note</span> SQL Client only supports connecting to the [REST Endpoint]({{< ref "docs/dev/table/sql-gateway/rest" >}}#rest-api) since version v2.
```